doc/nrf/ug_creating_matter_accessory.rst

```diff
@@ -0,0 +1,274 @@
+.. ug_creating_matter_accessory: 
+
+Creating Matter accessory
+#########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+
+| This tutorial shows how to create a custom device application using Matter.     
+| It will show a way to modify Matter Template sample in order to obtain a sensor application.
+
+The goal of this guide is to create a sensor, which, if active, periodically updates measured value.
+
+
+Matter Template sample
+**********************
+
+| The base for buiding a sensor device will be a Matter Template sample which provides a basic implementation of a Matter accessory.
```

Comment:
Maybe it's worth adding something like:
"Before proceeding, make sure that you can build and test the Matter Template sample with no issues".